eul/Store/PreferenceStore.swift

```diff
@@ -58,6 +58,11 @@ class PreferenceStore: ObservableObject {
     @Published var checkStatusItemVisibility = true
     @Published var isUpdateAvailable: Bool? = false
     @Published var checkUpdateFailed = true
+    @Published var appearanceMode = Preference.appearance.auto {
+        didSet {
+            changeColorScheme()
+        }
+    }
```

Comment:
maybe we can leverage Combine to listen to the changes of `preferenceStore.appearanceMode` in `StatusBarManager`? like what i did with `activeCancellable`

Reply:
Fixed